os/board/imxrt1020-evk/Kconfig

```diff
@@ -70,31 +35,6 @@ config IMXRT_AUTOMOUNT_SSSRW_MOUNTPOINT
 		Specifies the mount point where secure storage
 		will be mounted at.
 
-config IMXRT_AUTOMOUNT_ROMFS
-	bool "Automount romfs partiton"
-	default y
-	depends on IMXRT_AUTOMOUNT
-	depends on FS_ROMFS
-	---help---
-		If enabled, romfs partition will be mounted automatically
-		at boot.
-
-config IMXRT_AUTOMOUNT_ROMFS_DEVNAME
-	string "Device name of the partition for romfs file system"
-	default "/dev/mtdblock9"
-	depends on IMXRT_AUTOMOUNT_ROMFS
-	---help---
-		Specifies the device name (/dev/mtdblock9) of the partition
-		for romfs file system.
-
-config IMXRT_AUTOMOUNT_ROMFS_MOUNTPOINT
-	string "Mountpoint of the partition for romfs file system"
-	default "/rom"
-	depends on IMXRT_AUTOMOUNT_ROMFS
-	---help---
-		Specifies the mount point where romfs file system
-		will be mounted at.
-
 config IMXRT_AUTOMOUNT_TZDEVNAME
```

Comment:
For next step, please check this to merge into new common function.

Reply:
I will check and make another PR if needed.